<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>LibraryWindow</class>
 <widget class="QWidget" name="LibraryWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>876</width>
    <height>600</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>876</width>
    <height>600</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Library</string>
  </property>
  <!-- Left Side GroupBox for Models -->
  <widget class="QGroupBox" name="groupBox">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>50</y>
     <width>261</width>
     <height>531</height>
    </rect>
   </property>
   <property name="title">
    <string>Models</string>
   </property>
   <widget class="QPushButton" name="pushButton">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>20</y>
      <width>261</width>
      <height>32</height>
     </rect>
    </property>
    <property name="text">
     <string>Select All</string>
    </property>
   </widget>
   <widget class="QListWidget" name="listWidget">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>60</y>
      <width>261</width>
      <height>421</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="generateReport">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>490</y>
      <width>261</width>
      <height>32</height>
     </rect>
    </property>
    <property name="text">
     <string>Generate Report</string>
    </property>
   </widget>
  </widget>
  <!-- Model Details GroupBox -->
  <widget class="QGroupBox" name="modelDetailsGroupBox">
   <property name="geometry">
    <rect>
     <x>270</x>
     <y>10</y>
     <width>601</width>
     <height>100</height>
    </rect>
   </property>
   <property name="title">
    <string>Model Details</string>
   </property>
   <widget class="QLabel" name="modelTitleLabel">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>20</y>
      <width>200</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>Title:</string>
    </property>
   </widget>
   <widget class="QLabel" name="modelAuthorLabel">
    <property name="geometry">
     <rect>
      <x>220</x>
      <y>20</y>
      <width>200</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>Author:</string>
    </property>
   </widget>
   <widget class="QLabel" name="modelFilePathLabel">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>50</y>
      <width>400</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>File Path:</string>
    </property>
   </widget>
   <widget class="QLabel" name="modelThumbnailLabel">
    <property name="geometry">
     <rect>
      <x>420</x>
      <y>20</y>
      <width>100</width>
      <height>80</height>
     </rect>
    </property>
   </widget>
  </widget>
  <!-- Previews ListWidget -->
  <widget class="QListWidget" name="listWidgetPage">
   <property name="geometry">
    <rect>
     <x>270</x>
     <y>120</y>
     <width>601</width>
     <height>461</height>
    </rect>
   </property>
   <property name="viewMode">
    <enum>QListView::IconMode</enum>
   </property>
   <property name="iconSize">
    <size>
     <width>72</width>
     <height>72</height>
    </size>
   </property>
   <property name="gridSize">
    <size>
     <width>144</width>
     <height>108</height>
    </size>
   </property>
   <property name="movement">
    <enum>QListView::Static</enum>
   </property>
   <property name="resizeMode">
    <enum>QListView::Adjust</enum>
   </property>
   <property name="layoutMode">
    <enum>QListView::Batched</enum>
   </property>
   <property name="batchSize">
    <number>10</number>
   </property>
  </widget>
  <!-- Navigation and Search -->
  <widget class="QWidget" name="leftLayoutWidget">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>251</width>
     <height>34</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="leftHorizontalLayout">
    <item>
     <widget class="QPushButton" name="allLibraries">
      <property name="text">
       <string>All Libraries</string>
      </property>
      <property name="flat">
       <bool>true</bool>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLabel" name="label">
      <property name="text">
       <string>&lt;</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLabel" name="currentLibrary">
      <property name="font">
       <font>
        <bold>true</bold>
       </font>
      </property>
      <property name="text">
       <string>Current Library</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="rightLayoutWidget">
   <property name="geometry">
    <rect>
     <x>530</x>
     <y>10</y>
     <width>331</width>
     <height>32</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="rightHorizontalLayout">
    <item>
     <widget class="QComboBox" name="searchDropDown">
      <property name="enabled">
       <bool>false</bool>
      </property>
      <item>
       <property name="text">
        <string>Search Entire Library</string>
       </property>
      </item>
     </widget>
    </item>
    <item>
     <widget class="QLineEdit" name="searchInput">
      <property name="enabled">
       <bool>false</bool>
      </property>
      <property name="text">
       <string>Search Not Available</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
